src/components/Checkout/SocialShareV2.vue

```diff
@@ -0,0 +1,556 @@
+<template>
+	<section class="share hide-for-print">
+		<h2 class="share__headline" data-testid="share-headline">
+			Help {{ selectedLoan.name }} spread the word.
+			<p class="share__small__message">
+				{{ shareSubtitle }}
+			</p>
+		</h2>
+
+		<div class="share__social social">
+			<a
+				data-testid="share-facebook-button"
+				class="social__btn social__btn--facebook"
+				:href="facebookShareUrl"
+				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-Facebook-share']"
+			>
+				<kv-icon name="facebook-round" title="Facebook" class="social__icon" />
+				<span>Share</span>
+			</a>
+			<a
+				data-testid="share-twitter-button"
+				class="social__btn social__btn--twitter"
+				:href="twitterShareUrl"
+				target="_blank"
+				rel="noopener"
+				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-Twitter-share']"
+				@click="$showTipMsg('Thanks for tweeting!')"
+			>
+				<kv-icon name="twitter" title="Twitter" class="social__icon" />
+				<span>Tweet</span>
+			</a>
+			<a
+				data-testid="share-linkedin-button"
+				class="social__btn social__btn--linkedin"
+				:href="linkedInShareUrl"
+				target="_blank"
+				rel="noopener"
+				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-LinkedIn-share']"
+				@click="$showTipMsg('Thanks for sharing to LinkedIn!')"
+			>
+				<kv-icon name="linkedin" title="LinkedIn" class="social__icon" />
+				<span>Share</span>
+			</a>
+			<button
+				data-testid="share-copy-link-button"
+				class="social__btn social__btn--link tw-text-link tw-border-tertiary tw-border"
+				:class="copyStatus.class"
+				:disabled="copyStatus.disabled"
+				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-Copy-link-share']"
+				@click="copyLink"
+			>
+				<kv-icon name="clipboard" class="social__icon" />
+				<span>{{ this.copyStatus.text }}</span>
+			</button>
+		</div>
+	</section>
+</template>
+
+<script>
+import _map from 'lodash/map';
+import { orderBy } from 'lodash';
```

Comment:
Use `import orderBy from 'lodash/orderBy';` instead, because that will help with keeping the overall bundle size low.

Reply:
done!